dataset/histogram.cpp

```diff
@@ -118,13 +118,23 @@ Dim edge_dimension(const DataArrayConstView &a) {
   return *dims.begin();
 }
 
-/// Return true if the data array respresents a histogram for given dim.
-bool is_histogram(const DataArrayConstView &a, const Dim dim) {
+namespace {
+template <typename T> bool is_histogram_impl(const T &a, const Dim dim) {
   const auto dims = a.dims();
   const auto coords = a.coords();
-  return dims.contains(dim) && coords.contains(dim) &&
+  return dims.count(dim) == 1 && coords.contains(dim) &&
```

Comment:
More for my understanding than anything else. Why has this changed from `dims.contains` to `dims.count` here? From https://github.com/scipp/scipp/blob/master/core/include/scipp/core/dimensions.h#L94 isn't count just wrapping contains anyway?

Reply:
Never mind, just noticed Simon's comment above.